### DISCUSSION TITLE
Issue with dressing test script in the phy-robo-care branch.

### DISCUSSION THREAD

**Discussion by rohanb2018** [empriselab/RCareWorld#90] [Help - Using]
Hello, there appears to be an issue with the `templates/test_dressing.py`  script in the `phy-robo-care` branch of this repo. The observed behavior is that the cloth is on the rack, but the robot does not move. From debugging with @alkdischen's help, it seems like the issue is in the following line, which hangs:

```env = RCareWorld(executable_file="Dressing/DressingPlayer.x86_64")```

So there may be something wrong with that Unity executable file.

**Comment by shxingch** [comment]
The problem has now been resolved, and the scene file has been re-uploaded to GitHub! 
https://github.com/empriselab/RCareWorld/commit/38825fb03705daedca304b72ebd1054de918261f
